linkerd/app/outbound/src/http/logical.rs

```diff
@@ -157,17 +165,16 @@ impl<E> Outbound<E> {
                         // Sets the per-route response classifier as a request
                         // extension.
                         .push(classify::NewClassify::layer())
-                        .push_map_target(Logical::mk_route)
                         .push_on_service(http::BoxResponse::layer())
+                        // TODO(ver): CloneBoxService to flatten the stack type?
```

Comment:
this is released now fwiw: https://docs.rs/tower/0.4.11/tower/util/struct.BoxCloneService.html